docs/changehistory/NextVersion.md

```diff
@@ -639,3 +640,34 @@ class MyElement extends Element {
   ...
 }
 ```
+
+## Attach/detach db
+
+Allow the attachment of an ECDb/IModel to a connection and running ECSQL that combines data from both databases.
+
+Example of attaching a snapshot to a master file and running a query that combines data from both databases:
+```ts
```

Comment:
Please consider moving this example code to example-code-snippets and linking to it from here and more importantly from the `attachDb` method's API docs.

Reply:
I have added snippet and link it to NextVersion.md & attach/detach function api doc.